### PR TITLE
Cb 4417 migrate to reshadow user profile version update plugin tools plugins

### PR DESCRIPTION
**Pull request by sergeyteleshev**

### CODE REVIEW

File: webapp/packages/core-ui/src/Tabs/Tab/Tab.tsx [hunk 1]
```diff
@@ -54,7 +66,7 @@ export const Tab = observer<TabProps>(function Tab({ tabId, title, menuContext,
             disabled={disabled}
             onClick={tab.handleOpen}
           >
-            <tab-container>{children}</tab-container>
+            <tab-container className={сontainerClassName}>{children}</tab-container>
```
Comment (Wroud) [inline webapp/packages/core-ui/src/Tabs/Tab/Tab.tsx]: Please don't use props for specific inner elements, we will rewrite it to css modules with SContext in the future. Also currently please avoid to change anything related to tabs (we will migrate them all together later)

Comment (sergeyteleshev) [reply webapp/packages/core-ui/src/Tabs/Tab/Tab.tsx]: Okay I will revert that part

File: webapp/packages/plugin-user-profile/src/UserProfileForm/UserForm.m.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+.statusMessage {
+  composes: theme-typography--caption from global;
+  height: 24px;
+  padding: 0 16px;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+
+  & IconOrImage {
```
Comment (devnaumov) [inline webapp/packages/plugin-user-profile/src/UserProfileForm/UserForm.m.css]: .iconOrImage

File: webapp/packages/plugin-user-profile/src/UserProfileForm/UserForm.m.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    height: 24px;
+    width: 24px;
+  }
+}
+
+.flexBox {
```
Comment (devnaumov) [inline webapp/packages/plugin-user-profile/src/UserProfileForm/UserForm.m.css]: Probably we should give it a different name, .box or something like that.
Also we have a component with similar styles `<Container vertical overflow  />`, we could use this

**Review by devnaumov — CHANGES_REQUESTED** [review]

**Review by devnaumov — APPROVED** [review]

**Pull request merged**